Documentation/Contribution/Index.rst

```diff
@@ -0,0 +1,25 @@
+.. include:: /Includes.rst.txt
+
+============
+Contribution
+============
+
+Feel free to submit any pull request, or add documentation, tests, as you
+please. We will publish a new version every once in a while, depending on the
+amount of changes and pull requests submitted.
+
+License
+=======
+
+The extension is published under GPL v2+, all included third-party libraries are
+published under their respective licenses.
+
+Authors
+=======
+
+Over 100 contributors have been working on this area while this functionality
+was part of the TYPO3 Core. This package is now maintained by a loose group of
+TYPO3 enthusiasts inside the TYPO3 Community. Feel free to contact `Benni Mack`_
```

Comment:
Please avoid pointing to Benni here, refer to the issue tracker instead...

Reply:
Done.